digital-workspace/latest/support/index.md

```diff
@@ -1,7 +1,7 @@
 ---
 title: Supported Platforms
```

Comment:
I think this file is missing the ACS 7.2 addition - unless it's in another branch?

Reply:
I think I've updated it appropriately now for as 7.2.